pkg/dryrun/dryrun.go

```
@@ -636,20 +700,22 @@ func (d *DryRunner) saveOrPrintComplianceMessages(
+		return nil
+	}
 
+	f, err := os.Create(d.messagesPath)
+	if err != nil {
+		return err
+	}
+
+	for _, messages := range complianceMessages {
 		for _, msg := range messages {
-			cmd.Println(msg)
+			fmt.Fprintln(f, msg)
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintln` is not checked

(errcheck)